Makefile

```diff
@@ -294,6 +294,14 @@ container-health-check-image:
 appmesh-plugin-validator:
 	$(MAKE) -C misc/appmesh-plugin-validator $(MFLAGS)
 
+firelens-fluentbit-test-image:
```

Comment:
Does the fluentbit functional test get skipped on arm?

Reply:
yes it's already skipped on arm